schedule/yam/test_cases/verify_consistency_self_update.yaml

```diff
@@ -0,0 +1,9 @@
+name: verify_consistency_self_update
+description:    >
+    This test the consistency of self updates.
```

Comment:
"Verify consistency of a given self update channel by ..." Then you can describe briefly that we add the repo to a GA system, zypper dup and open YaST control center to ensure libraries used by YaST are working fine, but I recommend you to use the language used in the bug (better than mine :) )

Reply:
ok, done